.coveragerc

```diff
@@ -1,11 +1,17 @@
 [run]
 parallel = True
-include = src/*
+source =
+    globus_cli
+
+[paths]
+source =
+    src
+    */site-packages
```

Comment:
I was just putting in a PR ( https://github.com/globus/globus-sdk-python/pull/670 ) which contained something similar, but more complex than this. I like this better; I'll test and switch to match.